api-admin/src/main/java/bio/terra/pearl/api/admin/controller/dataimport/DataImportController.java

```diff
@@ -38,10 +38,42 @@ public ResponseEntity<Void> delete(
     return ResponseEntity.noContent().build();
   }
 
+  @Override
+  public ResponseEntity<Void> delete_0(
```

Comment:
I think you made the right choice to not make a separate ImportItem controller for this method.  Let's rename this method to deleteItem for clarity